db_assessment/0_configure_op_env.sh

```diff
@@ -0,0 +1,40 @@
+# This file configures the environemt for loading data files from the client
+# Edit this file and set the project name, data set name, and data set location to
+# where you want the data loaded.
+# Ensure you have proper access to the project and rights to create a data set.
+
+# This is the name of the project into which you want to load data
+export PROJECTNAME=yourProjectNameHere
```

Comment:
I don't think we want the user to actually edit this script.  What if we instead have it check to see if the variable is set and then fail if it's not?  In the instructions we can mention that you need to run the above export commands.

Reply:
Modified to check env var is set rather than expecting user to edit the file.